src/app/src/api/submissions.js

```diff
@@ -0,0 +1,48 @@
+import api from './api';
```

Comment:
This file is just here for reference at the moment. It does not reflect actual API routes.